docs/reference/commandline/node_rm.md

```diff
@@ -30,6 +31,24 @@ Example output:
     $ docker node rm swarm-node-02
     Node swarm-node-02 removed from swarm
 
+Removes nodes from the swarm that are in the down state. Attempting to remove
+an active node will result in an error:
+
+```bash
+$ docker node rm swarm-node-03
+Error response from daemon: rpc error: code = 9 desc = node swarm-node-03 is not down and can't be removed
```

Comment:
Yes, that's a known problem; we mark them as `bash` to make the highlighting more predictable, but basically many of our examples are combined "bash" / "shell" and "output". The output should just be plain text.

We tried putting it in a separate code-block but that's equally bad (they appear as two separate blocks in the layout).

*Removing* the bash hint is not solution; doing so will make the highlightjs "guess" what language it should pick; and in this case, it thinks it's Ruby, which looks even worse:

<img width="838" alt="screen shot 2016-08-02 at 22 39 07" src="https://cloud.githubusercontent.com/assets/1804568/17344441/5d38e960-5902-11e6-996e-e5571b0f488d.png">

In this case; changing to formal English (cannot) perhaps? :smile: